File: HireFlow.Backend/HireFlow.Application/Jobs/Common/JobDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HireFlow.Application.Jobs.Common
{
    public record JobDto(
        Guid Id,
        Guid RecruiterId,
        string Title,
        string? Description,
        string? WorkMode,
        string Status
    );

}

File: hireflow-client/src/app/domain/dtos/job.dto.ts
export interface JobDto{
    id : string,
    recruiterId : string,
    title : string,
    description : string,
    workMode : string,
    status : string   // 'Draft', 'Published', 'Closed'
}

export interface CreateJobRequest{
    title : string,
    description? : string,
    workMode? : string
}

export interface updateJobRequest{
    title : string,
    description? : string,
    workMode? : string
}